framework/src/Volo.Abp.IdentityModel/Volo/Abp/IdentityModel/AbpIdentityClientOptions.cs

```diff
@@ -8,5 +13,30 @@ public AbpIdentityClientOptions()
         {
             IdentityClients = new IdentityClientConfigurationDictionary();
         }
+
+        public IdentityClientConfiguration GetClientConfiguration(ICurrentTenant currentTenant, string identityClientName = null)
+        {
+            if (identityClientName.IsNullOrWhiteSpace())
+            {
+                identityClientName = IdentityClientConfigurationDictionary.DefaultName;
+            }
+
+            if (currentTenant.Id.HasValue)
+            {
+                var tenantConfiguration = IdentityClients.FirstOrDefault(x => x.Key == $"{identityClientName}.{currentTenant.Id}");
+                if (tenantConfiguration.Key == null && !currentTenant.Name.IsNullOrWhiteSpace())
```

Comment:
What if `tenantConfiguration` is null? FirstOrDefault returns null.

Reply:
The type of `tenantConfiguration `  is` KeyValuePair<string, IdentityClientConfiguration>`, which is `struct`.